.bazelrc.common

```diff
@@ -78,7 +80,8 @@ test:debug --test_output=streamed --test_strategy=exclusive --test_timeout=9999
 # The node process will break before user code starts and wait for the debugger to connect.
 run:debug --define=VERBOSE_LOGS=1 -- --node_options=--inspect-brk
 # The following option will change the build output of certain rules such as terser and may not be desirable in all cases
-build:debug --compilation_mode=dbg
+# It will also output both the repo cache and action cache to a folder inside the repo
+build:debug --compilation_mode=dbg --show_result=1 --disk_cache=bazel-cache/disk-cache --repository_cache=bazel-cache/repository-cache
```

Comment:
Should this be `build:debug --compilation_mode=dbg --show_result=1 --disk_cache=bazel/cache/disk --repository_cache=bazel/cache/repository` to coincide with the prefix change?

Reply:
YES! Good Catch @tylersmalley

Reply:
Done